Content.Client/StatusIcon/StatusIconSystem.cs

```diff
@@ -58,9 +58,8 @@ public List<StatusIconData> GetStatusIcons(EntityUid uid)
         if (!Exists(uid) || Terminating(uid))
             return new();
 
-        var ev = new GetStatusIconsEvent(new());
```

Comment:
Don't pass the uid, just use the event recipient. Unless this has to be turned into a broadcast event for whatever reason?

Reply:
Without the uid here the event subscription only fires without the uid and component parameters:
![image](https://github.com/space-wizards/space-station-14/assets/7510010/4b16985e-2d98-4ca7-992f-d034643ebd4f)

Reply:
This queries entities and what icons to show on them.

Reply:
I'm talking about not passing the Uid to the constructor, not the `RaiseLocalEvent()`, that does need the uid